packages/accounts-controller/CHANGELOG.md

```diff
@@ -7,6 +7,18 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.1.0]
+
+### Added
+
+- Add `listMultichainAccounts` action ([#4426](https://github.com/MetaMask/core/pull/4426))
```

Comment:
Actions are always preceded by a name, so the action is called `AccountsController:listMultichainAccounts` (this is what you'd pass to `messenger.subscribe`, in other words). Should this entry reflect that?

``` suggestion
- Add `AccountsController:listMultichainAccounts` action ([#4426](https://github.com/MetaMask/core/pull/4426))